### ISSUE TITLE
reading slides for week 1

### ISSUE THREAD

**Issue by melody1126** [UChicago-Computational-Content-Analysis/Frequently-Asked-Questions#4]
will the reading slides due week 1 (jan 14) be on the orienting reading? 

on Canvas week 1, there is a comment "Check out this video that walks through the Week 1 code and homework."
Is there a video we can watch to get a sense of the homework?

**Comment by jacyanthis** [comment]
I don't think there are any reading slides due today for any students. I think we just do those starting in the second week. James will talk about the Week 1 readings (Evans and Aceves 2016, Grimmer et al 2022) today to provide a more structured introduction to the course than student presentations would be. Does that make sense?

I think that "video" comment is extraneous—last year, we had prerecorded videos. I've deleted that comment, and I'm sorry for the inconvenience.

**Comment by melody1126** [comment]
> 

Thank you for clarifying Jacy! Just to confirm - only students who signed up to present need to submit the reading or coding slides that week?

**Comment by jacyanthis** [comment]
That's correct, @melody1126. I believe Canvas may give you some annoying notifications like "assignment missing" if you didn't submit slides, but you can ignore those for all the weeks you did not sign up to present. Also, as noted in the [sign-up spreadsheet](https://docs.google.com/spreadsheets/d/1bi45HS39BRx_x2tltwWfBwakbioc4pXnO_2uKqlgTVA/), keep in mind that code presentations are done the week before the code is due, and reading presentations are due the week the readings are listed in the syllabus.

This will all become a lot clearer in class today.
